docs/aws-config.md

```diff
@@ -149,6 +149,119 @@ Specify the following values in the example files:
 
     * Replace `<YOUR_STORAGE_CLASS_NAME>` with `gp2`. This is AWS's default `StorageClass` name.
 
+## ALTERNATIVE: Setup permissions using kube2iam
+
+[Kube2iam](https://github.com/jtblin/kube2iam) is a Kubernetes application that allows managing AWS IAM permissions for pod via annotations rather than operating on API keys.
+
+> This path assumes you have `kube2iam` already running in your Kubernetes cluster. If that is not the case, please install it first, following the docs here: https://github.com/jtblin/kube2iam
+
+It can be set up for Ark by creating a role that will have required permissions, and later by adding the permissions annotation on the ark deployment to define which role it should use internally.
+
+1. Create a Trust Policy document to allow the role being used for EC2 management & assume kube2iam role:
+
+    ```bash
+    cat > heptio-ark-role-trust-policy.json <<EOF
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "Service": "ec2.amazonaws.com"
+                },
+                "Action": "sts:AssumeRole"
+            },
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::<AWS_ACCOUNT_ID>:role/<ROLE_CREATED_WHEN_INITIALIZING_KUBE2IAM>"
+                },
+                "Action": "sts:AssumeRole"
+            }
+        ]
+    }
+    EOF
+    ```
+
+2. Create the IAM role:
+
+    ```bash
+    aws iam create-role --role-name heptio-ark-role --assume-role-policy-document file://./heptio-ark-role-trust-policy.json
```

Comment:
Here you have `file://./....` whereas below you have `file://....` - would it make sense to be consistent?

Reply:
Fixed